cedar-lean/Cedar/Thm/Data/Set.lean

```diff
@@ -38,6 +38,22 @@ theorem if_wellformed_then_exists_make [LT α] [DecidableLT α] (s : Set α) :
   intro h₁
   exists s.elts
 
+theorem wf_iff_sorted {α} [LT α] [DecidableLT α] [StrictLT α]  (s : Set α) :
+  s.WellFormed ↔ s.elts.Sorted
+:= by
+  simp only [WellFormed, make, toList, elts]
+  have hid : (fun x => x) = @id α := by
+    apply funext ; simp only [id_eq, implies_true]
+  rw [hid]
+  constructor <;>
+  intro h
```

Comment:
In other places (like line 926 above) you have this on one line.  Which is preferable?

Reply:
No thought out preference though empirically, it appears that I favor putting things on line :D